app/star-rating.sass

```diff
@@ -9,4 +9,32 @@ h1
    margin: 0
 
 #app
-   font-size: 3em
\ No newline at end of file
+   font-size: 3em
+
+.star-rating
+    display: inline-block
+    unicode-bidi: bidi-override
+    direction: rtl
+    border-width: 0px
+
+    .star-rating__clicked::before,
+    .star-rating__clicked ~ .star-rating__label:before
+        font-family: "Font Awesome 5 Free"
+        font-weight: 900
+        content: "\f005"
+        color: rgba(255, 255, 0, 100)
+
+    .star-rating__label::before
+        font-family: "Font Awesome 5 Free"
+        font-weight: 400
+        content: "\f005"
+
+    .star-rating__label:hover::before,
+    .star-rating__label:hover ~ .star-rating__label:before
+        font-family: "Font Awesome 5 Free"
+        font-weight: 900
+        content: "\f005"
+        color: rgba(255, 255, 150, 100)
+
+    .star-rating__input
+        opacity: 0
```

Comment:
Prefer `display: none`. Having `opacity: 0` means that the element is rendered and can be clicked on. Try clicking in between the stars a bit to see how they can be selected even when the star is not being clicked.

Reply:
see commit "removes display of radio buttons"